components/ButtonNavBar/ButtonNavBar.js

```diff
@@ -29,10 +34,19 @@ export default function ButtonNavBar({ id, deleteCard }) {
     });
   }
 
+  function handleToggle(event) {
+    event.preventDefault();
+    event.stopPropagation();
+    toggleMastered(id);
+  }
+
   return (
     <StyledButtonNavBar>
       <CardButton onClick={onEdit}>Bearbeiten</CardButton>
       <CardButton onClick={handleDelete}>Löschen</CardButton>
+      <CardButton onClick={handleToggle}>
```

Comment:
```suggestion
      <CardButton onClick={() => onToggle(id)}
}>
```

Reply:
Why ist the function written as arrow-function?